cluster/config-defaults.yaml

```diff
@@ -140,8 +141,15 @@ tokeninfo_url: "" # can be set when local tokeninfo is disabled
 skipper_local_tokeninfo: "bridge"
 {{end}}
 
+# oauth2 UI login - grant flow
+skipper_oauth2_ui_login: "true"
+skipper_oauth2_secrets_dir: /etc/skipper/oauth
+skipper_oauth2_secret_file: TODO KMS stups
```

Comment:
This should have a reasonable default (for e2e or pet clusters).

Reply:
was fixed